src/components/Header.js

```diff
@@ -5,7 +5,12 @@ export default function Header() {
 		<header>
 			<nav>
 				<Link href="/">Home</Link>
+				<br />
+				<Link href="/main">Form</Link>
+				<br />
+
```

Comment:
```suggestion
```

We always try to get gaps between different elements through styling instead of the `<br>`element